org.eclipse.xtext/META-INF/MANIFEST.MF

```diff
@@ -8,49 +8,57 @@ Bundle-Vendor: Eclipse Xtext
 Bundle-Localization: plugin
 Bundle-RequiredExecutionEnvironment: JavaSE-11
 Export-Package: org.eclipse.xtext,
- org.eclipse.xtext.build;x-friends:="org.eclipse.xtext.testing,
+ org.eclipse.xtext.build;
+  x-friends:="org.eclipse.xtext.testing,
```

Comment:
is this default formatting`?

Reply:
I guess so: I added them from the MANIFEST editor and that's the result.